utils/src/test/scala/cromwell/pipeline/utils/configs/LogInfoUtilsTest.scala

```diff
@@ -0,0 +1,114 @@
+package cromwell.pipeline.utils.configs
+
+import com.typesafe.config.{ Config, ConfigFactory }
+import cromwell.pipeline.utils.ApplicationConfig
+import org.scalatest.Matchers._
+import org.scalatest.WordSpec
+
+class LogInfoUtilsTest extends WordSpec {
+
+  val testConfigSource: String =
+    s"""webservice {
+       |  interface = "localhost"
+       |  port = 8080
+       |}
+       |auth {
+       |  secretKey = "testSecretAuthKey"
+       |  hmacAlgorithm = "HS256"
+       |  expirationTimeInSeconds {
+       |    accessToken = 100
+       |    refreshToken = 200
+       |    userSession = 5000
+       |  }
+       |}
+       |database {
+       |  postgres_dc {
+       |    profile = "testProfile"
+       |    db {
+       |      properties {
+       |        serverName = "testServer"
+       |        portNumber = 1234
+       |        databaseName = "testDatabaseName"
+       |        user = "testUser"
+       |        password = "testPostgrePassword"
+       |      }
+       |      numThreads = 10
+       |    }
+       |  }
+       |  gitlab {
+       |    url = "http://gitlab.com"
+       |    token = "testGitLabToken"
+       |    defaultFileVersion = "testVersion"
+       |    defaultBranch = "testBranch"
+       |  }
+       |  mongo {
+       |    user = "testMongoUser"
+       |    password = "testMongoPassword"
+       |    host = "testMongoHost"
+       |    port = 1
+       |    authenticationDatabase = "testAuthDatabase"
+       |    database = "testMongoDatabase"
+       |    collection = "testMongoCollection"
+       |  }
+       |}""".stripMargin
+
```

Comment:
Is it necessary to check the whole thing? Or you can test just some of lines?

Reply:
Thats a good question. The function tested is strictly tied with our current configuration structure, and I couldn't find a way to avoid creating full-size dummies.

Reply:
Honestly, it's an implementation flaw :) We should be able to pretty print every single configuration class separately. For now I think it's ok but keep in mind that on the real project most likely you would have to make it possible to test every config separately
Of course it makes sense to test them as a whole too.

Reply:
Maybe I should return to that issue in a month then and see if I can do it better with a new experience :)